vivisect/__init__.py

```diff
@@ -247,7 +248,7 @@ def delExtension(self, name):
         '''
         self._extensions.pop(name, None)
 
-    def getVivGuid(self):
+    def getVivGuid(self, generate=True):
```

Comment:
Is this parameter useful now that things have been simplified down?

Reply:
less so.  but since the function is entitled "getVivGuid()" it still seems like we should have the option to *not* generate one if one does not exist.